sechub-doc/src/docs/asciidoc/documents/techdoc/09_04_howto_quick_test_created_sechub_server_release_local.adoc

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+[[section-howto-quck-test-server-release-local]]
+=== Howto quickly test a sechub server release on local machine
+
+For a "quick test" we start the release simply in integrationtest mode.
+
+==== Get the release
+
+You got different options: You can either download an existing release from https://github.com/Daimler/sechub/releases ,
+or you build it from scratch. 
+
+[NOTE]
+====
+Here an example for a scratch-build: Checked out gi tag `v0.19.1-server` and by executing `./gradlew bootJar`) you will
+find the server release 0.19.1 at `sechub-server/build/libs/sechub-server-0.19.1.jar` 
+====
+
+==== Start release at your machine  
```

Comment:
I believe **on** your machine is more accurate.

Reply:
Done